Logic/PaymentActions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Logic.SpacePark;
+using SpaceApi.Models;
+using System;
+using System.Linq;
+
+namespace Logic
+{
+    public class PaymentActions
+    {
+        public void Pay(Parking parking)
+        {
+            Console.Clear();
+            Console.WriteLine("Using swish to pay...");
+
+            Payment pay = new Payment {Amount = parking.Fee, User = parking.ParkedBy, PayDate = DateTime.UtcNow};
+            var spaceApi = new SpaceParkApi();
+            spaceApi.AddPay(pay);
+
+            Console.WriteLine("Payment successful");
+            Console.WriteLine($"Payment By: {pay.User}. Amount: {pay.Amount} credits. Payment Date: {pay.PayDate}");
+            Console.ReadKey();
+        }
+        public void Receipts()
+        {
+            string name = StandardMessages.NameReader();
+            var spaceApi = new SpaceParkApi();
+            var payments = spaceApi.GetReceipts(name);
+
+            if (payments.Result != null)
+            {
+                if (!payments.Result.Any())
+                {
+                    Console.WriteLine("No receipts found.");
+                }
+                else
+                {
+                    foreach (var r in payments.Result)
+                    {
+                        Console.WriteLine($"Payment By: {r.User}. Amount: {r.Amount} credits. Payment Date: {r.PayDate}");
+                    }
+                }
+            }
+            else
+            {
+                StandardMessages.NotAllowedMessage();
+            }
```

Comment:
Rather then having nested if stmts, and a for each loop inside.
A sln to make it more readable would be to

```suggestion
if (payments.Result != null)
{
  StandardMessages.NotAllowedMessage();
  return;
}

if (!payments.Result.Any())
{
  Console.WriteLine("No receipts found.");
  return;
}

for each (var r in payments.Result)
{
Console.WriteLine($"Payment By: {r.User}. Amount: {r.Amount} credits. Payment Date: {r.PayDate}");
}
```

**Disclaimer:** _Suggestion is written without intelissense, please bare in mind._